vue-components/tests/unit/compositions/validateable.ts

```diff
@@ -0,0 +1,29 @@
+import { getFeedbackTypeFromProps, errorProp } from '../../../dist/components/util/ValidationMessageComposition';
```

Comment:
Forgot to add to to the review above the path here should be to `@/compositions/validateable`

Reply:
Good catch! I will fix it right away

Reply:
Wait, `components/util/ValidationMessageComposition` doesn't exist anymore in this commit. How did this even work and pass CI? :upside_down_face:

Reply:
Oy vay, I think it's because dist is still cached...

Reply:
Mh, we may want to look into that again. But this code should be fixed now